pkg/operator/controller/ingress/deployment.go

```diff
@@ -435,16 +439,28 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 
 	routeAdmission := operatorv1.RouteAdmissionPolicy{
 		NamespaceOwnership: operatorv1.StrictNamespaceOwnershipCheck,
+		WildcardPolicy:     operatorv1.WildcardPolicyDisallowed,
 	}
-	if admission := ci.Spec.RouteAdmission; admission != nil && len(admission.NamespaceOwnership) > 0 {
-		routeAdmission.NamespaceOwnership = ci.Spec.RouteAdmission.NamespaceOwnership
+	if admission := ci.Spec.RouteAdmission; admission != nil {
+		if len(admission.NamespaceOwnership) > 0 {
+			routeAdmission.NamespaceOwnership = ci.Spec.RouteAdmission.NamespaceOwnership
+		}
+		if len(admission.WildcardPolicy) > 0 {
+			routeAdmission.WildcardPolicy = ci.Spec.RouteAdmission.WildcardPolicy
+		}
 	}
 	switch routeAdmission.NamespaceOwnership {
 	case operatorv1.StrictNamespaceOwnershipCheck:
 		env = append(env, corev1.EnvVar{Name: "ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK", Value: "false"})
 	case operatorv1.InterNamespaceAllowedOwnershipCheck:
 		env = append(env, corev1.EnvVar{Name: "ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK", Value: "true"})
 	}
+	switch routeAdmission.WildcardPolicy {
+	case operatorv1.WildcardPolicyDisallowed:
```

Comment:
Should we have a `default:` case that either replaces or does a `fallthrough` to this case?

Reply:
done